src/deserialize_sequence.cc

```diff
@@ -340,65 +342,27 @@ Step deserialize_step(const std::filesystem::path& path)
 std::istream& operator>>(std::istream& stream, Sequence& seq)
 {
     std::string line;
-    std::string step_setup_script{};
-    bool has_step_setup_script = false;
-    bool load_step_setup_script = false;
+    std::string step_setup_script;
 
     while(std::getline(stream, line, '\n'))
-    {
-        auto tmp = gul14::trim_sv(line);
-        if (not load_step_setup_script)
-        {
-            if (not has_step_setup_script and tmp.find_first_of("-- step setup: [[[[[") != std::string::npos)
-            {
-                tmp.remove_prefix(std::string("-- step setup [[[[[").size());
-
-                if (not tmp.empty())
-                    step_setup_script += tmp;
-
-                has_step_setup_script = true;
-
-                continue;
-            }
-            else if (has_step_setup_script)
-            {
-                if(tmp.find_last_of("]]]]]") != std::string::npos)
-                {
-                    tmp.remove_suffix(std::string("]]]]]").size());
-
-                    if (not tmp.empty())
-                        step_setup_script += tmp;
-
-                    has_step_setup_script = false;
-                    load_step_setup_script = true;
-                }
-                else
-                {
-                    std::cout << '\'' << line << '\'' << std::endl;
-                    step_setup_script += line + '\n';
-                }
-
-                continue;
-            }
-        }
-    }
+        step_setup_script += (gul14::trim_right(line) + '\n');
 
-    if (has_step_setup_script and not load_step_setup_script)
-        throw Error(gul14::cat("Sequence '", seq.get_label(), "' has a misconfigured "
-            "step setup script configuration"));
+    if (not step_setup_script.empty())
+    {
+        // remove the previous trailing line feed
+        step_setup_script.pop_back();
```

Comment:
Ok. Got it. Because of the wrong use of `gul14::trim_right_sv()` in `Sequence::set_step_setup_script()` (see [here](https://github.com/taskolib/taskolib/pull/49/files#r1040671078)) related to the removing of the last newline in the above code snippet, the test case unfortunately succeeded.

I now removed the claimed line and add a new test case for checking the setter/getter method in `Sequence`. Please have a look (final test case in `test_Sequence.cc`).